plugins/transcend/transcend-nvme.h

```diff
@@ -5,17 +5,19 @@
 #if !defined(TRANSCEND_NVME) || defined(CMD_HEADER_MULTI_READ)
 #define TRANSCEND_NVME
 
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
+
 #include "cmd.h"
  
 
 PLUGIN(NAME("transcend", "Transcend vendor specific extensions", NVME_VERSION),
     COMMAND_LIST(
-			ENTRY("healthvalue", "NVME health percentage", getHealthValue)
-			ENTRY("badblock", "Get NVME bad block number", getBadblock)
- 
+        ENTRY("healthvalue", "NVME health percentage", get_health_value)
+        ENTRY("badblock", "Get NVME bad block number", get_bad_block)
+        ENTRY("plphealthvalue", "Get NVME PLP Health.", get_plp_health)
     )
 );
 
-#endif
+#endif /* TRANSCEND_NVME */
 
-#include "define_cmd.h"
+#include "define_cmd.h"
```

Comment:
Should be added a new line code on the end of the line (before the end of file).